application/helpers/expressions/em_manager_helper.php

```diff
@@ -6985,15 +6953,26 @@ public static function GetRelevanceAndTailoringJavaScript($bReturnArray = false)
                     }
                 }
             }
+            $pageRelevanceInfo[] = array(
```

Comment:
the «global» page relevance info is set at end with last group seq. Maybe out of group seq too ?

Reply:
Why is this record needed? It is not related to any question id. Is it for handling EM at group level?